bin/docker/waltz-server.sh

```diff
@@ -5,19 +5,25 @@ cmd=$1
 
 imageSource=waltz-server:distDocker
 imageName=com.wepay.waltz/waltz-server
-containerName=waltz-server
+containerName="$2_server"
 networkName=waltz-network
-ports=55180-55182:55180-55182
+configFolder="$2"
+
+if [ $cmd = "start" ]; then
+    portsOccupiedLowerBound=$3
```

Comment:
nit: May be "portsLowerBound" instead of "portsOccupiedLowerBound" ? Similarly, at other places too.